README.md

```diff
@@ -703,43 +703,34 @@ When building oneMKL the SYCL implementation can be determined, by setting the `
-enable_mklcpu_backend    | ENABLE_MKLCPU_BACKEND    | True, False         | True
-enable_mklgpu_backend    | ENABLE_MKLGPU_BACKEND    | True, False         | True
-*Not Supported*          | ENABLE_CUBLAS_BACKEND    | True, False         | False
-*Not Supported*          | ENABLE_CUSOLVER_BACKEND  | True, False         | False
-*Not Supported*          | ENABLE_CURAND_BACKEND    | True, False         | False
-*Not Supported*          | ENABLE_NETLIB_BACKEND    | True, False         | False
-*Not Supported*          | ENABLE_ROCBLAS_BACKEND   | False               | False
-enable_mklcpu_thread_tbb | ENABLE_MKLCPU_THREAD_TBB | True, False         | True
-build_functional_tests   | BUILD_FUNCTIONAL_TESTS   | True, False         | True
-build_doc                | BUILD_DOC                | True, False         | False
-target_domains (list)    | TARGET_DOMAINS (list)    | blas, lapack, rng   | All domains
-
-**ONEMKL_SYCL_IMPLEMENTATION=hipsycl**
-Conan Option | CMake Option | Supported Values | Default Value
- :---------- | :----------- | :--------------- | :---
-build_shared_libs        | BUILD_SHARED_LIBS        | True, False         | True
-enable_mklcpu_backend    | ENABLE_MKLCPU_BACKEND    | True, False         | True
-enable_mklgpu_backend    | ENABLE_MKLGPU_BACKEND    | False               | False
-*Not Supported*          | ENABLE_CUBLAS_BACKEND    | False               | False
-*Not Supported*          | ENABLE_CURAND_BACKEND    | False               | False
-*Not Supported*          | ENABLE_NETLIB_BACKEND    | True, False         | False
-*Not Supported*          | ENABLE_ROCBLAS_BACKEND   | True, False         | False
-enable_mklcpu_thread_tbb | ENABLE_MKLCPU_THREAD_TBB | True, False         | True
-build_functional_tests   | BUILD_FUNCTIONAL_TESTS   | True, False         | True
-build_doc                | BUILD_DOC                | True, False         | False
-target_domains (list)    | TARGET_DOMAINS (list)    | blas                | All domains
-N/A                      | HIPSYCL_TARGETS          | depends on target device | none
+Conan Option | CMake Option | Supported Values | Default Value | Supported Compilers
+ :---------- | :----------- | :--------------- | :---          | :---
+build_shared_libs        | BUILD_SHARED_LIBS        | True, False         | True      | All
+enable_mklcpu_backend    | ENABLE_MKLCPU_BACKEND    | True, False         | True      | All
+enable_mklgpu_backend    | ENABLE_MKLGPU_BACKEND    | True, False         | True      | All
+*Not Supported*          | ENABLE_CUBLAS_BACKEND    | True, False         | False     | LLVM
+*Not Supported*          | ENABLE_CUSOLVER_BACKEND  | True, False         | False     | LLVM
+*Not Supported*          | ENABLE_CURAND_BACKEND    | True, False         | False     | LLVM
+*Not Supported*          | ENABLE_NETLIB_BACKEND    | True, False         | False     | All
+*Not Supported*          | ENABLE_ROCBLAS_BACKEND   | False               | False     | hipSYCL
```

Comment:
Supported Values should be True, False, right?

Reply:
yes...